_posts/0000-01-02-Dev-Xagon.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
# So this is a joke I love: 3 frogs on a lake, first say quack, second quack quack and the third quack quack quack. The frist one pulls a gun and shoots frog 3. The second asks the first "Why would you do such a thing?" "*In mafia voice* "He knew too much" the first frog said